Fix terminal workspace trust bypass

Adds missing return statements after _onProcessExit calls in the terminal
trust and empty-workspace-cwd checks, preventing continued execution of
_createProcess after the terminal should have been terminated.

Remove when Code-OSS is updated to >= 1.132.1.

@backported: https://github.com/microsoft/vscode/commit/3154a68fc151bcafe371f89d197412645e6a7616
@finding-id: CVE-2026-69278 GHSA-h9j4-x76r-fvj4
Index: b/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts
===================================================================
--- a/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts
+++ b/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts
@@ -1581,11 +1581,13 @@ export class TerminalInstance extends Di
 		// Allow remote and local terminals from remote to be created in untrusted remote workspace
 		if (!trusted && !this.remoteAuthority && !this._workbenchEnvironmentService.remoteAuthority) {
 			this._onProcessExit({ message: nls.localize('workspaceNotTrustedCreateTerminal', "Cannot launch a terminal process in an untrusted workspace") });
+			return;
 		} else if (this._workspaceContextService.getWorkspace().folders.length === 0 && this._cwd && this._userHome && normalizeDriveLetter(this._cwd) !== normalizeDriveLetter(this._userHome)) {
 			// something strange is going on if cwd is not userHome in an empty workspace
 			this._onProcessExit({
 				message: nls.localize('workspaceEmptyCreateTerminalCwd', "Cannot launch a terminal process in an empty workspace with cwd {0} different from userHome {1}", this._cwd, this._userHome)
 			});
+			return;
 		}
 		// Re-evaluate dimensions if the container has been set since the xterm instance was created
 		if (this._container && this._cols === 0 && this._rows === 0) {
